cpp/include/cudf/copying.hpp

```diff
@@ -19,18 +19,18 @@
 #include "cudf.h"
 #include "types.hpp"
 
-namespace cudf {
-namespace experimental {
-
 /** ---------------------------------------------------------------------------*
 * @brief Indicates when to allocate a mask, based on an existing mask.
 * ---------------------------------------------------------------------------**/
-enum mask_allocation_policy {
+enum class  mask_allocation_policy {
     NEVER, ///< Do not allocate a null mask, regardless of input
     RETAIN, ///< Allocate a null mask if the input contains one
     ALWAYS ///< Allocate a null mask, regardless of input
 };
 
+namespace cudf {
```

Comment:
Keep mask_allocation_policy in the cudf::experimental namespace. The rest looks good